packages/muelu/src/Transfers/Smoothed-Aggregation/MueLu_SaPFactory_def.hpp

```diff
@@ -60,6 +60,7 @@
 #include "MueLu_SingleLevelFactoryBase.hpp"
 #include "MueLu_TentativePFactory.hpp"
 #include "MueLu_Utilities.hpp"
+#include "Tpetra_Details_Behavior.hpp"
```

Comment:
This is problematic, as we don't usually expose Tpetra/Epetra specific headers in MueLu.